packages/wallet/src/contract-tests/test-utils.ts

```diff
@@ -109,7 +109,7 @@ export async function concludeGame(provider: ethers.providers.JsonRpcProvider, c
     channel,
     allocation: fiveFive,
     destination: [participantA.address, participantB.address],
-    turnNum: 6,
+    turnNum: 7,
```

Comment:
Double check that `yarn test:ci:contracts` still passes with this change. I had to handle this in #161

Reply:
It does